src/sidebar/services/test/session-test.js

```diff
@@ -172,30 +172,26 @@ describe('SessionService', () => {
 
       it('should cache the returned profile data', () => {
         const session = createService();
-        return session
-          .load()
-          .then(() => {
-            return session.load();
-          })
-          .then(() => {
-            assert.calledOnce(fakeApi.profile.read);
-          });
+        session.load();
+        session.load();
+
+        assert.calledOnce(fakeApi.profile.read);
       });
 
       it('should eventually expire the cache', () => {
         clock = sinon.useFakeTimers();
         const CACHE_TTL = 5 * 60 * 1000; // 5 minutes
 
-        const session = createService();
-        return session
-          .load()
-          .then(() => {
-            clock.tick(CACHE_TTL * 2);
-            return session.load();
-          })
-          .then(() => {
-            assert.calledTwice(fakeApi.profile.read);
-          });
+        try {
+          const session = createService();
+          session.load();
+          clock.tick(CACHE_TTL * 2);
+          session.load();
+        } finally {
+          clock.restore();
```

Comment:
There is an `afterEach` block which restores the clock after each test, so this is not needed.

Reply:
When a fake clocks is used on a single test, I think it is better to declare and restore the fake clock within the function. I have removed the block-level variable. I think this is beneficial from a readability perspective.